src/main/java/taxi/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -32,14 +31,14 @@ public Manufacturer create(Manufacturer manufacturer) {
             }
             return manufacturer;
         } catch (SQLException ex) {
-            throw new DataProcessingException("Can't insert into DB", ex);
+            throw new DataProcessingException("Couldn't add manufacturer into DB ", ex);
         }
     }
 
     @Override
     public Optional<Manufacturer> get(Long id) {
         String query = "SELECT * FROM manufacturer "
-                + "WHERE manufacturer_id = ? and isDeleted = false ";
+                + "WHERE manufacturer_id = ? and deleted = false ";
         Manufacturer manufacturer = null;
         try (Connection connection = ConnectionUtil.getConnection()) {
             PreparedStatement preparedStatement = connection.prepareStatement(query);
```

Comment:
https://github.com/AntonPanchishko/ApplicationHW/pull/6#discussion_r557536408 this relates to all your statements